README.md

```diff
@@ -62,6 +63,23 @@ _For mimicking an email inbox (ala mailtrap) for local usage._
   * `MAIL_PASSWORD=null`
   * `MAIL_ENCRYPTION=null`
 
+### MinIO
+_For running an S3 service locally._
+
+* Edit the `/etc/hosts` file on your system and make a reference for `127.0.0.1 s3.docker`
+* Open the admin console http://s3.docker:10001 and login.
+  * username: root
+  * password: password
+* Create an appropriate bucket for each project as needed.
+* Modify the project's `.env` as follows:
+  * `FILESYSTEM_DRIVER=s3`
+  * `AWS_ACCESS_KEY_ID=root`
+  * `AWS_SECRET_ACCESS_KEY=password`
+  * `AWS_DEFAULT_REGION=us-east-1`
+  * `AWS_BUCKET=[project-bucket-name]`
```

Comment:
Not sure if DevopsToolKit is the right place for this, or if it'd be better in project initial setup or something once this gets added to template, but it'd be nice if this could spin up the initial bucket based on the env configuration.

Reply:
I've set it up like that for previous projects, so if that's a direction we want to move, it shouldn't be an issue.